src/components/structures/SearchBox.js

```diff
@@ -21,7 +21,7 @@ import { _t } from '../../languageHandler';
 import { KeyCode } from '../../Keyboard';
 import sdk from '../../index';
 import dis from '../../dispatcher';
-import rate_limited_func from '../../ratelimitedfunc';
+import { debounce } from 'lodash';
```

Comment:
I thought `throttle` closer matched what we had? Or is there a reason to prefer the different behaviour (I think thge difference being whether it keep executing the thing every n seconds on sustained activity).

Reply:
From reading the documentation, debounce seems like the thing we want here with the trailing edge later defined. If someone were to roll their face on the keyboard, we probably want to debounce that rather than rate limit it.

Reply:
Indeed, what Travis said. Feels faster in practice as well, as the delay can be smaller.